### PR TITLE
Phase2-hgx325J Remove reference to 6-inch wafers in 5 classes of Validation/HGCalValidation

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Remove reference to 6-inch wafers in 5 classes of Validation/HGCalValidation

#### PR validation:

Use the runTheMatrix test workflows

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by emanueleusai** [comment]
are the changes in HGCalHitCalibration, e.g. https://tinyurl.com/2g85rrfc understood? This seems to be different from related PR #39517 where previously empty plots became populated.

**Comment by bsunanda** [comment]
@emanueleusai I believe the changes provided in HGCalHitCalibration are indeed justified. The earlier code was missing HE silicon part

**Comment by emanueleusai** [comment]
+1

File: Validation/HGCalValidation/plugins/HGCalHitCalibration.cc [hunk 1]
```diff
@@ -292,10 +271,8 @@ void HGCalHitCalibration::analyze(const edm::Event& iEvent, const edm::EventSetu
         DetId hitid = (it_haf.first);
         // dump raw RecHits and match
         if (rawRecHits_) {
-          if ((hitid.det() == DetId::Forward &&
-               (hitid.subdetId() == HGCEE || hitid.subdetId() == HGCHEF || hitid.subdetId() == HGCHEB)) ||
-              (hitid.det() == DetId::HGCalEE) || (hitid.det() == DetId::HGCalHSi) || (hitid.det() == DetId::HGCalHSc) ||
-              (hitid.det() == DetId::Hcal && hitid.subdetId() == HcalEndcap))
+          if ((hitid.det() == DetId::Forward) || (hitid.det() == DetId::HGCalEE) || (hitid.det() == DetId::HGCalHSi) ||
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Validation/HGCalValidation/plugins/HGCalHitCalibration.cc:274]: This is for HFNose which uses Forward in the det() - maybe later we shall check on sundered as well

**Comment by perrotta** [comment]
+1

**Pull request merged**